### ISSUE TITLE
Something went worng while trying to use react-accordion-with-richtext

### ISSUE THREAD

**Issue by PicGadm** [pnp/sp-dev-fx-webparts#3072]
### Disclaimer

Yes

### Sample

https://github.com/pnp/sp-dev-fx-webparts/tree/main/samples/react-accordion-with-richtext

### Author(s)

@Arun Kumar Perumal

### What happened?

We are trying to use the samples, we were able to build and package the react-accordion-with-richtext to an sppkg file and uploaded it to our appcat.

We published the sample and added the webpart to a page, there we got an "Something went wrong".

![MicrosoftTeams-image (5)](https://user-images.githubusercontent.com/113907481/196738651-5bc811f8-0a93-4027-a3b1-210475f10bda.png)

### Steps to reproduce

1. Install the needed NodeJS version for this sample
2. Open CMD
3. Navigate to the folder location of the sample
4. Check which version of SPFX is needed using "npm ls --depth=0 -g"
5. Uninstall the wrong version if needed "npm uninstall @microsoft/generator-sharepoint@<version>"
6. Install the wrong version if needed "npm install @microsoft/generator-sharepoint@1.13.1 --save --save-exact"
7. "npm install"
8. "gulp clean"
9. "gulp bundle --ship"
10. "gulp package-solution --ship"
11. Go to the sharepoint app catalog
12. Upload the sppkg file from "<samplepath>\sharepoint\solution"
13. Choose to deploy to all sites in the organization
14. Open a page and add the webpart
15. Now we have the error

### Expected behavior

We expect to see the webpart, not the error.

### Target SharePoint environment

SharePoint Online

### Developer environment

Windows

### Browsers

- [ ] Internet Explorer
- [X] Microsoft Edge
- [ ] Google Chrome
- [ ] FireFox
- [ ] Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [ ] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Paste the results of SPFx doctor

The command "m365 spfx doctor" only gives me documentation about how to use the SPFX doctor. What command do you need me to execute?

**Comment by Abderahman88** [comment]
Hi @PicGadm,

You can run `m365 spfx doctor` directly. Have you installed [CLI for M365](https://pnp.github.io/cli-microsoft365/) ?

![image](https://user-images.githubusercontent.com/36161889/197410131-8528b620-5873-4182-83ff-c7cbb8823c6b.png)

I've tried the sample and had no issues.

![image](https://user-images.githubusercontent.com/36161889/197410289-a3b0cf87-5d83-41b1-97bc-6e1b96f49737.png)

**Comment by PicGadm** [comment]
> Hi @PicGadm,
> 
> You can run `m365 spfx doctor` directly. Have you installed [CLI for M365](https://pnp.github.io/cli-microsoft365/) ?
> 
> ![image](https://user-images.githubusercontent.com/36161889/197410131-8528b620-5873-4182-83ff-c7cbb8823c6b.png)
> 
> I've tried the sample and had no issues.
> 
> ![image](https://user-images.githubusercontent.com/36161889/197410289-a3b0cf87-5d83-41b1-97bc-6e1b96f49737.png)

Yes I have installed this CLI module.
When I run the command I get this message.

![2022-10-24_09h17_20](https://user-images.githubusercontent.com/113907481/197469429-ee80be6e-f888-46b8-867b-3eddfca6391a.png)

**Comment by PicGadm** [comment]
@hugoabernier This is the case I was talking about in https://github.com/pnp/sp-dev-fx-webparts/issues/2413.
I find it harsh that you tell us that you don't support issues?

We try to follow the guidelines to build the code provided here, resulting in a not working solution.
I did not ask anyone to provide the .sppkg file. 
I am asking help to build it correctly, that's why these issues on github exist not?

So I am kindly asking some help. The error tells me "Failed to load URL 'https://publiccdn.sharepointonline.com/picanolgroup.sharepoint.com/sites/......".

I need to be able to remove the part "publiccdn.sharepointonline.com/" from that url I guess because the file exists at "https://picanolgroup.sharepoint.com/sites/...".

I tried to find that location in the code sample to remove that part and then I saw the "write-manifest.json" in the config folder, this contains "cdnBasePath": "", I tried to change that, resulting in the same error, same url, nothing changed.

So can anyone help a guy out that is trying to understand these samples and wants to learn?
